recipe/meta.yaml

```diff
@@ -59,3 +59,4 @@ extra:
     - jakirkham
     - scopatz
     - ocefpaf
+    - pkgw
```

Comment:
Since you just added yourself you will need someone else to merge this.  Feel free to ping me once Travis-CI finishes.